### PR TITLE
ping goal owners

### PR DESCRIPTION
**Pull request by nikomatsakis**
Add a zulip command to ping goal owners. I intend this to run once a week eventually but right now it's still executed manually.

### CODE REVIEW

File: src/zulip.rs [hunk 1]
```diff
@@ -188,6 +189,12 @@ fn handle_command<'a>(
                                 .await
                                 .map_err(|e| format_err!("Failed to await at this time: {e:?}"))
                             }
+                            Some("ping-goals") => {
```
Comment (jackh726) [inline src/zulip.rs:192]: So...I feel like it may be prudent to gate this so that only you (@nikomatsakis) can run this.

**Review by jackh726 — COMMENTED** [review]
Mostly good, one thought.

File: src/zulip.rs (hunk 1 above)
Comment (nikomatsakis) [reply src/zulip.rs:192]: ...good idea, at least for now. :)

**Comment by nikomatsakis** [comment]
@jackh726 added the simplest of all possible ACLs :)

File: src/handlers/project_goals.rs [hunk 2]
```diff
@@ -39,6 +39,18 @@ impl Job for ProjectGoalsUpdateJob {
     }
 }
 
+/// Returns true if the user with the given github id is allowed to ping all group people
+/// and do other "project group adminstrative" tasks.
+pub async fn check_project_goal_acl(_gh: &GithubClient, gh_id: u64) -> anyhow::Result<bool> {
+    /// Github ID of the user allowed to ping all group people.
+    ///
+    /// FIXME: We should create a team for the person/people managing the goals program
+    /// and check that the zulip person is on it, but I'm too
```
Comment (jackh726) [inline src/handlers/project_goals.rs:48]: Left out a word here, but its funnier this way

**Review by jackh726 — APPROVED** [review]

**Pull request merged**